public/sass/_variables.dark.generated.scss

```diff
@@ -102,7 +102,7 @@ $text-color: rgb(204, 204, 220);
 $text-color-semi-weak: rgba(204, 204, 220, 0.65);
 $text-color-weak: rgba(204, 204, 220, 0.65);
 $text-color-faint: rgba(204, 204, 220, 0.6);
-$text-color-emphasis: #fff;
+$text-color-emphasis: #FFFFFF;
```

Comment:
Is this on purpose?

Reply:
oh no...autogenerated when I started Grafana.

Reply:
probably should be committed to the repo then. i guess someone has forgotten? 🤔

Reply:
yeah...I think so. Seems to be our preferred style of writing hex code.